### PR TITLE
Add deprecation notices to Dask chart

### PR DESCRIPTION
**Pull request by jacobtomlinson**
We are migrating the Dask community helm charts over to our own repository that will be registered on Helm Hub as per the instructions https://helm.sh/blog/intro-helm-hub/.

This PR adds deprecation notices to the chart in stable and points users to the new repository.

Please let me know if there is anything else I need to change here.

cc @mrocklin

### CODE REVIEW

**Comment by zanhsieh** [comment]
@jacobtomlinson 
Could you sign DCO and bump the chart version to latest for one last time please?

File: stable/dask/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v1
 name: dask
-version: 3.1.0
+version: 3.1.1
```
Comment (davidkarlsen) [inline stable/dask/Chart.yaml:4]: please add the deprecation flag as well: https://github.com/helm/charts/blob/master/PROCESSES.md
```suggestion
version: 3.1.1
deprecated: true
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by jacobtomlinson** [comment]
Done @davidkarlsen. I hadn't seen that process in the docs, I think I've followed it correctly now.

**Comment by jacobtomlinson** [comment]
Yeah seems a bit weird. I've updated to reflect that anyway.

**Comment by jacobtomlinson** [comment]
Changes have been made @davidkarlsen. Can I get another review on this?

**Comment by jacobtomlinson** [comment]
We've moved over to developing on our own hosted chart so please could we get this reviewed and merged so that folks know this one is deprecated?

File: stable/dask/Chart.yaml [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 appVersion: 1.1.5
-description: Distributed computation in Python with task scheduling
+# The foo chart is deprecated and no longer maintained. For details deprecation,
```
Comment (scottrigby) [inline stable/dask/Chart.yaml]:
```suggestion
# This chart is deprecated and no longer maintained. For details deprecation,
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
One more change, otherwise looks good

**Comment by jacobtomlinson** [comment]
Thanks @scottrigby. Changes made.

**Review by davidkarlsen — APPROVED** [review]

**Comment by jacobtomlinson** [comment]
🎉

**Pull request merged**